server/src/metadata/field-metadata/field-metadata.service.ts

```diff
@@ -14,22 +17,23 @@ export class FieldMetadataService {
   ) {}
 
   public async createFieldMetadata(
-    name: string,
+    displayName: string,
     type: string,
     objectId: string,
     workspaceId: string,
   ): Promise<FieldMetadata> {
     return await this.fieldMetadataRepository.save({
-      displayName: name,
+      displayName: displayName,
       type,
       objectId,
       isCustom: true,
-      targetColumnName: generateColumnName(name),
+      targetColumnName: generateColumnName(displayName), // deprecated
```

Comment:
@Weiko I guess we can't hard drop this, as we don't have anything implemented on the front part for now ?
cc @charlesBochet

Reply:
if you meant "can" then yes I guess we can drop this part instead of note as "deprecated" however the entity-resolver part is not working with targetColumnMap yet and still uses targetColumnName so I didn't want to introduce breaking changes. I was planning to remove the code once it's not used anymore @magrinj

Reply:
Oh ok I understand ! :)